nebula-components/components/_tab.scss

```diff
@@ -0,0 +1,30 @@
+@if $nb-use-tab {
+  .#{$nb-namespace}c-tabs {
+    background-color: $nb-tabs-background;
+  }
+
+  .#{$nb-namespace}c-tab {
+    padding: $nb-tab-padding;
+    color: $nb-tab-color;
+    background-color: $nb-tab-background;
+
+    &:focus {
+      outline: none;
```

Comment:
I'd argue that we want to keep this for accessibility - keyboard navigation,
We should also provide hover state.  We can be clever and remove the outline when we are hovered and focused because only a mouse can hover so not affecting keyboard only user.